paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'CLOVER: Community-scale sustainable energy system modelling'
+tags:
+ - Python
+ - energy access
+ - minigrid
+ - renewable energy
+ - sustainable development
+authors:
+ - name: Philip Sandwell
+  orcid: 0000-0003-1117-5095
+  affiliation: "1, 2"
+  corresponding: true
+- name: Benedict Winchester
+  orcid: 0000-0002-2880-1243
+  affiliation: "2, 3"
+- name: Hamish Beath
+  orcid: 0000-0002-5124-9143
+  affiliation: "1, 2"
+- name: Jenny Nelson
+  orcid: 0000-0003-1048-1330
+  affiliation: "1, 2"
+affiliations:
+ - name: Department of Physics, Blackett Laboratory, Imperial College London, SW7 2AZ, United Kingdom
+   index: 1
+ - name: Grantham Institute - Climate Change and the Environment, Imperial College London, SW7 2AZ, United Kingdom
+   index: 2
+ - name: Department of Chemical Engineering, Imperial College London, SW7 2AZ, United Kingdom
+   index: 3
+date: 19 August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Sustainable Development Goal 7 aims to provide sustainable, affordable, reliable and modern energy access to all by 2030. In order for this goal to be achieved, sustainable energy interventions in developing countries must be supported with design tools which can evaluate the technical performance of energy systems as well as their economic and environmental impacts.
```

Comment:
Provide context maybe:
The United Nations Sustainable Development Goals (SDGs) were published in 20XX [CITATION]. SDG 7 aims to ..."

Reply:
I think it's okay to jump straight to SDG7, I would have thought that it would be assumed knowledge for anyone reading the paper. @hamishbeath what do you think?

Reply:
I agree with you Phil, I think just SDG7 here, but perhaps reference it?

Reply:
I've added a reference as suggested